com.github.simonwhitaker.Gibo.appdata.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
```

Comment:
Is this a command-line utility? If so, it's not `desktop-application`, but rather `console-application`.

Reply:
Done